components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1101,14 +1137,20 @@ bool ConversationDriver::HasPendingConversationEntry() {
 }
 
 int ConversationDriver::GetContentUsedPercentage() {
-  if (GetCurrentModel().max_page_content_length >
+  auto& model = GetCurrentModel();
+  if (!model.option->is_leo_model()) {
```

Comment:
nit: more readable to use `is_custom_model()` instead of `!is_leo_model()`

Reply:
revised this condition